server/ce2e/iframe_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build exclude
```

Comment:
Temporarily disabled due to flakiness. Will discuss with @jespino, but maybe there's a way we can consolidate our testing approaches.

Reply:
We should probably move the embedded feature into its own plugin.